Modules/Axis/Sources/Axis/Map/Map.swift

```diff
@@ -804,23 +804,22 @@ extension Map : CustomStringConvertible {
 
              "\u{2028}": "\\u2028",
              "\u{2029}": "\\u2029",
-
-             "\r\n": "\\r\\n"
         ]
 
         func escape(_ source: String) -> String {
-            var string = "\""
+            var string: String.UnicodeScalarView = "\""
 
-            for character in source.characters {
-                if let escapedSymbol = escapeMapping[character] {
-                    string.append(escapedSymbol)
+            source.unicodeScalars.forEach {
```

Comment:
I think its slightly better style to use `for scalar in source.unicodeScalars` here instead